packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Icon } from '@fluentui/react';
+import {
+  FileCard,
+  FileCardGroup,
+  truncatedFileName,
+  extension,
+  FileMetadata,
+  extractFileMetadata,
+  FileDownloadHandler
+} from './file-sharing';
+import React from 'react';
+import { ChatMessage } from '@internal/react-components';
+
+/**
+ * @beta
+ */
+export interface FileDownloadCards {
+  /**
+   * User id for the message
```

Comment:
This is actually the local participant's userId, right? You can get userId of the participant that sent a message from the message itself. You need the local userId so that `downloadHandler` can take care of auth, if any.

Reply:
User Id was supposed to be local participant's id, I have updated the comment